openbas-front/src/admin/components/settings/data_ingestion/XlsMapperPopover.tsx

```diff
@@ -78,6 +96,12 @@ const XlsMapperPopover: FunctionComponent<Props> = ({
           handleClose={handleCloseEdit}
         />
       </Drawer>
+      <DialogDuplicate
+        open={openDuplicate}
+        handleClose={handleCloseDuplicate}
+        handleSubmit={submitDuplicate}
+        text={`${t('Do you want to duplicate this XLS Mapper :')} ${mapper.import_mapper_name} ?`}
```

Comment:
```suggestion
        text={`${t('Do you want to duplicate this XLS mapper :')} ${mapper.import_mapper_name} ?`}
```
To stay consistent with the other messages :)